x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/synthetics/single_metric_config.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SYNTHETICS_STEP_NAME } from '../constants/field_names/synthetics';
+import { ConfigProps, SeriesConfig } from '../../types';
+import { FieldLabels, FORMULA_COLUMN } from '../constants';
+import { buildExistsFilter } from '../utils';
+
+export function getSyntheticsSingleMetricConfig({ dataView }: ConfigProps): SeriesConfig {
+  return {
+    defaultSeriesType: 'line',
+    xAxisColumn: {},
+    yAxisColumns: [
+      {
+        operationType: 'median',
```

Comment:
Should single metric types have an operationType applied?

Reply:
yes it does, we can display average, median or percentile values